lib/config.js

```diff
@@ -7,7 +7,12 @@ import path from 'path';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const cfg = config['account-http'] = {};
+const cfg = config['account-http'] = {
+  // account registration options
+  registration: {
+    authorizationRequired: 'turnstile'
```

Comment:
Does this require turnstile by default always?

Reply:
That is the new default, but it can be turned off in config.

Reply:
If we make this a new default, we would have to release this as breaking.  Not sure we wawnt to have something like this on by default in any case.

Reply:
Updated to not default to turnstile.